_includes/commands/table-cmds.md

```diff
@@ -4,6 +4,7 @@
 
 | Command                 | Description                                     |
 | ----------------------- | ----------------------------------------------- |
+| mcetableapplycellstyle | Applies the specified styles to the selected cells. The following styles can be changed with this command: `background-color`, `border-color`, `border-style`, and `border-width`. |
```

Comment:
We should use camel case for consistency here, eg: `mceTableApplyCellStyle`

Reply:
Fixed by: https://github.com/tinymce/tinymce-docs/pull/1570/commits/82e7d51f189765c65c7b9e80f3360d50a1dee960